website/source/docs/telemetry/index.html.md

```diff
@@ -109,6 +109,18 @@ when retrieving metrics using the above described signals.
     <td>Raft transactions / `interval`</td>
     <td>Counter</td>
   </tr>
+  <tr>
+    <td>`nomad.raft.lastIndex`</td>
+    <td>Index of the last log</td>
+    <td>Sequence number</td>
+    <td>Gauge</td>
+  </tr>
+  <tr>
+    <td>`nomad.raft.appliedIndex`</td>
+    <td>Index of the last applied log</td>
```

Comment:
~ Is it worth adding some context and/or the caveat in https://godoc.org/github.com/hashicorp/raft#Raft.AppliedIndex : 
> This is generally lagging behind the last index, especially for indexes that are persisted but have not yet been considered committed by the leader. NOTE - this reflects the last index that was sent to the application's FSM over the apply channel but DOES NOT mean that the application's FSM has yet consumed it and applied it to its internal state. Thus, the application's state may lag behind this index.